editor/editor_themes.cpp

```diff
@@ -1600,7 +1600,7 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_constant("shadow_offset_x", "RichTextLabel", 1 * EDSCALE);
```

Comment:
There are some inconsistencies where some times it is `1 * EDSCALE` and some just `EDSCALE`